src/sync/sys/windows/net.rs

```diff
@@ -0,0 +1,303 @@
+// Copyright (c) 2019 Ant Financial
```

Comment:
Seems need to update copyright?

Reply:
not sure what the policy is for this project?

Reply:
I think it'd make sense for @liubin's suggestion given this lives in the containerd org. The language shouldn't matter at all, but another data point for rust code is the rust-extensions repo follows the convention from the other projects and has the containerd authors copyrights for all of its crates.

Reply:
I pulled the content for `containerd/containerd` (same as rust-extensions).  Do all the mod.rs files need it too? Maybe we can follow up and change it all over with a CI check too?